README.md

```diff
@@ -7,8 +7,54 @@
 
 ## Instructions
 
-Ask the user to enter their age and display a greeting to them based on whether
-they are younger or older than 18. Take care to handle the case where they are
-exactly 18.
+Let's go back to our `Student` class example!
 
-Display an extra special message to them if they are younger than 10.
+```java
+public class Student {
+    private String name;
+    
+    Student(String name) {
+        this.name = name;
+    }
+    
+    public String getName() {
+        return name;
+    }
+    
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Take in a student numeric grade and provide feedback
+     * on how they did on a test
+     * @param grade : int - numeric grade the student received
+     *                on the test
+     */
+    public void calculateGrade(int grade) {
+        if (grade >= 70) {
+            System.out.println("Congrats! You passed!");
+            if (grade >= 90) {
+                System.out.println("Wow! You got an A!");
+            }
+        } else {
+            System.out.println("Oops! Better luck next time!");
+        }
+    }
+    
+    public static void main(String args[]) {
+        // Write Student user input here
+    }
+}
+```
+
+Let's ask the user for input and use our `Student` class! 
+
+First, we will prompt the user for their name and then create a `Student`
```

Comment:
Might be better to use imperative sentences so students know what is expected of them. For example, "First, prompt the user for their name and then create a `Student` instance with that name.".